### PR TITLE
Add ap1 to values.yaml

### PR DESCRIPTION
**Pull request by taka2noda**
#### What this PR does / why we need it:
What:
add ap1 to site at values.yaml
reference : https://docs.datadoghq.com/getting_started/site/

Why:
lack of ap1 info.

#### Which issue this PR fixes
N/A *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
This PR apply the same change at https://github.com/DataDog/helm-charts/pull/1079 against operator and op-worker values.yaml

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] `CHANGELOG.md` has been updated
- [x] Variables are documented in the `README.md`

### CODE REVIEW

**Comment by 001wwang** [comment]
Opened https://github.com/DataDog/helm-charts/pull/1082

**Comment by taka2noda** [comment]
Hi @001wwang , thank you for asking and I agree with your idea but,,, sorry for late response due to time zone(midnight zzz...)
I've merged your change into this PR. Could you check and merge this PR please?

File: charts/observability-pipelines-worker/values.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ datadog:
   ## Set to 'us3.datadoghq.com' to send data to the US3 site.
   ## Set to 'us5.datadoghq.com' to send data to the US5 site.
   ## Set to 'ddog-gov.com' to send data to the US1-FED site.
+  ## Set to 'ap1.datadoghq.com' to send data to the AP1 site.
```
Comment (clamoriniere) [inline charts/observability-pipelines-worker/values.yaml:36]: if a file is modified in a chart like here in `observability-pipelines-worker`, it is mandatory to also bump the chart version and add an entry in the corresponding changelog.
If the chart version is not updated, the new chart version and documentation will not be created.

**Review by clamoriniere — CHANGES_REQUESTED** [review]
Hi @taka2noda 

Thanks for opening this Pull Request. the changes are legit. 
Please follow the Pull request checklist to not miss one step in the pull request creation.

File: charts/observability-pipelines-worker/values.yaml (hunk 1 above)
Comment (001wwang) [inline charts/observability-pipelines-worker/values.yaml:36]: 🤦 Ahhh sorry, I had thought I'd picked this change into #1082 but I missed it somehow. Feel free to bump the chart to version `1.2.2` to include this doc change.

Comment (taka2noda) [reply charts/observability-pipelines-worker/values.yaml:36]: Ahhh @001wwang NP at all~ Got it! Thank you for supporting explanation. I'll bump the version and re-commit.

Comment (taka2noda) [reply charts/observability-pipelines-worker/values.yaml:36]: I've done. https://github.com/DataDog/helm-charts/pull/1081/commits/5fbe8e633d0a429bc30afc2cc6f03819bd2333d9

**Review by 001wwang — APPROVED** [review]
Thanks!

**Comment by taka2noda** [comment]
Hi @clamoriniere 

I've bumped the chart version and add an entry in the corresponding changelog.

However, the `go Test` end fail.
I've re-check my [change files ](https://github.com/DataDog/helm-charts/pull/1081/files) and check [the error log](https://github.com/DataDog/helm-charts/actions/runs/5257516667/jobs/9500406101) 

This error seems to occur [at `/home/runner/work/helm-charts/helm-charts/test/datadog-operator/baseline_test.go:111`](https://github.com/DataDog/helm-charts/actions/runs/5257516667/jobs/9500406101#step:6:17150)
but, I couldn't figuer out why it ends fail...

Could you help me to investigate the fail reason and suggest me neccesary steps, please?
Sorry take your time for such minor change.
If it is better to withdrow this change, please feel free to let me know.

Thank you.

**Review by clamoriniere — APPROVED** [review]

**Comment by taka2noda** [comment]
I've resolved conflict.

**Comment by levan-m** [comment]
Please run `make update-test-baselines` and include the manifest changes in the PR. This will resolve the test failure. 

Test was added recently and isn't properly documented. I'll update the checklist and readme.

**Comment by taka2noda** [comment]
Thank you @levan-m ! I've run and finally passed local `make test` and here!
 ```
Datadog-helm-charts on   add-ap1 !4 ❯ make test                                                                 at  12:15:39
go fmt -C test ./...
go vet -C test ./...
go test -C test ./... -count=1
?   	github.com/DataDog/helm-chart/test/common	[no test files]
ok  	github.com/DataDog/helm-chart/test/datadog-operator	5.828s
```

**Review by levan-m — APPROVED** [review]

**Comment by levan-m** [comment]
Approved. Could you please sign the commits before merging?

**Comment by taka2noda** [comment]
Hi @levan-m Thank you for pointing out the signature. 
In order to prevent miss changes from being imported by push -f, please let the approval be put back on pending until the changes are completed. 
I will inform you again when the change is complete. Thank you.

**Review by 001wwang — DISMISSED** [review]
Hey @taka2noda, we're going to release the observability-pipelines-worker chart shortly at `v1.3.0`. To avoid conflict, we're going to fold in these doc changes to our PR: https://github.com/DataDog/helm-charts/pull/1094. Please remove the `observability-pipelines-worker` changes from this PR.

File: charts/observability-pipelines-worker/values.yaml (hunk 1 above)
Comment (clamoriniere) [inline charts/observability-pipelines-worker/values.yaml:36]:
```suggestion
```

File: charts/observability-pipelines-worker/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Observability Pipelines Worker
 
-![Version: 1.2.1](https://img.shields.io/badge/Version-1.2.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1](https://img.shields.io/badge/AppVersion-1-informational?style=flat-square)
+![Version: 1.2.2](https://img.shields.io/badge/Version-1.2.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1](https://img.shields.io/badge/AppVersion-1-informational?style=flat-square)
```
Comment (clamoriniere) [inline charts/observability-pipelines-worker/README.md:3]:
```suggestion
![Version: 1.2.1](https://img.shields.io/badge/Version-1.2.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1](https://img.shields.io/badge/AppVersion-1-informational?style=flat-square)
```

File: charts/observability-pipelines-worker/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: observability-pipelines-worker
-version: "1.2.1"
+version: "1.2.2"
```
Comment (clamoriniere) [inline charts/observability-pipelines-worker/Chart.yaml:3]:
```suggestion
version: "1.2.1"
```

File: charts/observability-pipelines-worker/CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 # Changelog
 
+## 1.2.2
+
+* Add AP1 Site Comment in `values.yaml`.
+
```
Comment (clamoriniere) [inline charts/observability-pipelines-worker/CHANGELOG.md:6]:
```suggestion
```

File: charts/observability-pipelines-worker/CHANGELOG.md [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
 ## 1.2.1
 
 * Official image `1.2.1`
```
Comment (clamoriniere) [inline charts/observability-pipelines-worker/CHANGELOG.md:9]:
```suggestion
* Official image `1.2.1`
* Add AP1 Site Comment in `values.yaml`.
```

File: charts/datadog-operator/values.yaml [hunk 6]
```diff
@@ -23,6 +23,7 @@ appKey:  # <DATADOG_APP_KEY>
 ## Set to 'us3.datadoghq.com' to send data to the US3 site.
 ## Set to 'us5.datadoghq.com' to send data to the US5 site.
 ## Set to 'ddog-gov.com' to send data to the US1-FED site.
+## Set to 'ap1.datadoghq.com' to send data to the AP1 site.
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:26]:
```suggestion
```

**Comment by taka2noda** [comment]
Hi. I close this PR and re-open [the PR](https://github.com/DataDog/helm-charts/pull/1100)
Because I tried to revert and varify past commit... but there are several conflicts within the target commit and I want to avoide risk of unexpected marge due to such minor release.
So I'd like to close this commit and re-open the PR to add ap1.
Thank you for supporting this PR.

**Pull request closed without merging**